src/images/icons.json

```diff
@@ -192,6 +192,7 @@
     "phrase",
     "plus",
     "plus-heavy",
+    "plus-minus",
```

Comment:
can we not do this and instead use the icon from `phosphor`?

Reply:
It's being passed to `ActionFormRow` and it's used in all forms, so I think it will be good to refactor all icons used there but maybe in other PR since this one is getting messy?

Reply:
I think that component has support for passing in a rendered component now so we can try and see if it works
![image](https://github.com/JoinColony/colonyCDapp/assets/23449297/e4e4c14d-a03e-48b7-bc72-2792f1d8721b)

Reply:
It worked, so I've changed all icons in this table to phosphor